### PR TITLE
Jsinha/add rhel8

### PR DESCRIPTION
**Pull request by jaymalasinha**
## Description
Enable RHEL8 Packages

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by jaymalasinha** [comment]
https://buildkite.com/chef/chef-chef-master-omnibus-adhoc/builds/79 in progress to test the pipeline updates

File: lib/chef/provider/package/zypper.rb [hunk 1]
```diff
@@ -145,8 +145,10 @@ def zypper_package(command, global_options, *options, names, versions)
           zipped_names = zip(names, versions)
           if zypper_version < 1.0
             shell_out!("zypper", global_options, gpg_checks, command, *options, "-y", names)
+            sleep 120
           else
             shell_out!("zypper", global_options, "--non-interactive", gpg_checks, command, *options, zipped_names)
+            sleep 120
```
Comment (lamont-granquist) [inline lib/chef/provider/package/zypper.rb]: is this still just for testing?  that's a long time...

Comment (jaymalasinha) [reply lib/chef/provider/package/zypper.rb]: zypper process is locked by the previous test to we have seen it fail multiple times in the release builds, the sleep time allows the process to complete and unlock before the next verification step triggers

Comment (jaymalasinha) [reply lib/chef/provider/package/zypper.rb]: https://github.com/chef/chef/pull/8742 . in progress, good to merge once this passes

Comment (jaymalasinha) [reply lib/chef/provider/package/zypper.rb]: @lamont-granquist found a correct fix for zypper issue we are seeing on the SLES tester, will add that add it in a different PR.

**Comment by jaymalasinha** [comment]
Ready tp merge after https://buildkite.com/chef/chef-chef-master-omnibus-adhoc/builds/94 passes

**Comment by lamont-granquist** [comment]
merged ASAIWU (As Soon As I Wake Up)

**Pull request merged**